scripts/run_april_2026_attendance.py

```
@@ -0,0 +1,120 @@
+
+ROOT = Path(__file__).resolve().parent.parent
+sys.path.insert(0, str(ROOT))
+
+from triage.roster_parser import parse_roster, RosterParseError
+from triage.attendance_report import generate_attendance_report
+
+
+def _fmt(decimal_hours: float) -> str:
+    """Format decimal hours as HH:MM (e.g. 23.75 → '23:45')."""
+    hh = int(decimal_hours)
+    mm = int(round((decimal_hours - hh) * 60))
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)